BUILD

```diff
@@ -0,0 +1,93 @@
+Building eudev
+===================
+
+The following instructions provide the steps needed to acquire and build the eudev
+software on the Debian, Devuan, and compatible Linux distributions.
```

Comment:
I'd add something like `Similar instructions apply to other distributions, though, the package names differ.`

Reply:
Agreed. Added.